python/array_record_data_source_test.py

```diff
@@ -24,8 +24,8 @@
 from absl.testing import flagsaver
 from absl.testing import parameterized
 
-from python import array_record_data_source
-from python import array_record_module
+from array_record.python import array_record_data_source
+from array_record.python import array_record_module
```

Comment:
This test and the one above is executed by Bazel, rather than pytest after installing the package, so I think these paths are messed up a bit. It would be better to have them in pytest side, but for Bazel-managed tests I think we need to keep `from python ...`.